### PR TITLE
Add iOS support

### PR DESCRIPTION
**Pull request by vladvlasov256**
**Fixes**:
- added `ios ...` command,
- added SwiftUI templates for colors, fonts, and spacings,
- added snapshot tests
- added iOS specification to Readme

### CODE REVIEW

File: src/ios/formatters/__tests__/spacingFormatter.test.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  it('can handle dictionary with spacing tokens', () => {
+    const { spacingsTemplate } = TEMPLATES('', 'DesignSystem')
+
+    const spacingConfig: FormatterConfig = {
+      template: spacingsTemplate.source,
+      header: HEADER,
+      packageName: 'nl.elements.skeleton',
+    }
+
+    const properties = {
```
Comment (jeffreydelooff) [inline src/ios/formatters/__tests__/spacingFormatter.test.ts:19]: These could probably be shared between the tests. We could create a file with all the properties together.

File: src/ios/formatters/colorFormatter.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+      const toArgb = color => {
+        const colorStr = Color(color).toHex8()
+
+        return colorStr.slice(6) + colorStr.slice(0, 6)
+      }
+
+      const colors = tokens.map(({ name, value }) => ({
+        name,
+        value: toArgb(value),
```
[the remark this answers is not in the document]
Comment (vladvlasov256) [reply src/ios/formatters/colorFormatter.ts]: I have to use custom hex initializer that works with ARGB. But firstly I ignored the alfa-component.

Now `toArgb()` is removed, the custom initializer in the template is fixed.

File: src/ios/formatters/typographyFormatter.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import Color from 'tinycolor2'
```
[the remark this answers is not in the document]
Comment (vladvlasov256) [reply src/ios/formatters/typographyFormatter.ts]: Done

File: src/ios/formatters/__tests__/spacingFormatter.test.ts (hunk 1 above)
Comment (vladvlasov256) [reply src/ios/formatters/__tests__/spacingFormatter.test.ts]: Done

**Pull request merged**